packages/UI/src/components/Button/Button.tsx

```diff
@@ -1,116 +1,55 @@
 import styled, { css } from 'styled-components';
 
 interface ButtonProps {
-  bgColor?: string;
-  textColor?: string;
-  marginTop?: number;
-  width?: number;
-  fontSize?: number;
-  hero?: boolean;
-  modal?: boolean;
-  intro?: boolean;
+  buttonType?: string;
+  buttonSize?: string;
```

Comment:
this is good but can be improved and more explicit, rather than (any)string we need it to be certain strings